### PR TITLE
CPP-3658 SC docs moved to docs.sonarcloud.io

### PR DESCRIPTION
**Pull request by arseniy-sonar**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -30,12 +30,12 @@ You can take a look at the link:sonar-project.properties[sonar-project.propertie
...
 = Documentation
 
-- https://sonarcloud.io/documentation/analysis/languages/cfamily/[Documentation of the C, C++ and Objective-C plugin and its Build Wrapper]
+- https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/[Documentation of the C, C++ and Objective-C plugin and its Build Wrapper]
 - https://docs.travis-ci.com/user/sonarcloud/[Documentation of the SonarCloud Travis Add-on]
-- https://sonarcloud.io/documentation/getting-started/github/[Documentation of using SonarCloud with GitHub]
-- https://sonarcloud.io/documentation/analysis/languages/cfamily/#analysis-cache[Configuring C, C++ and Objective-C analysis cache]
-- https://sonarcloud.io/documentation/analysis/languages/cfamily/#multithreaded-code-scan[Configuring multithreaded execution]
-- https://sonarcloud.io/documentation/analysis/languages/cfamily/#analysis-steps-using-compilation-database[Using compilation database (compile_commands.json)]
+- https://docs.sonarcloud.io/getting-started/github/[Documentation of using SonarCloud with GitHub]
```
Comment (amelie-renard-sonarsource) [inline README.adoc:35]: I think we can remove this link as it is not related with the current example

**Review by amelie-renard-sonarsource — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (arseniy-sonar) [reply README.adoc:35]: IMO, it is still relevant, as it describes the GitHub integration (not necessarily GH Actions) and it mentions Travis CI in the "CI-based analysis" section.

Comment (amelie-renard-sonarsource) [reply README.adoc:35]: Ok then!

Comment (amelie-renard-sonarsource) [reply README.adoc:35]: So you can ignore the same comment I wrote in the other Travis example

**Review by amelie-renard-sonarsource — APPROVED** [review]

**Pull request merged**